src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -352,6 +352,52 @@ TEST_F(HMICapabilitiesTest, LoadCapabilitiesFromFile) {
   EXPECT_EQ("SE", vehicle_type_so["trim"].asString());
 }
 
+TEST_F(HMICapabilitiesTest,
+       HmiCapabilitiesInitialized_UiVrTtsIviNotCooperating) {
+  // Precondition
+  hmi_capabilities_test->set_is_vr_cooperating(false);
+  hmi_capabilities_test->set_is_tts_cooperating(false);
+
+  hmi_capabilities_test->set_is_ui_cooperating(false);
+  hmi_capabilities_test->set_is_navi_cooperating(false);
+  hmi_capabilities_test->set_is_ivi_cooperating(false);
+  EXPECT_TRUE(hmi_capabilities_test->is_hmi_capabilities_initialized());
+}
+
+TEST_F(HMICapabilitiesTest, HmiCapabilitiesInitialized) {
+  // Precondition
+  SetCooperating();
+  smart_objects::SmartObjectSPtr language(
+      new smart_objects::SmartObject(smart_objects::SmartType_Map));
+
+  EXPECT_CALL(*(MockMessageHelper::message_helper_mock()),
+              CreateModuleInfoSO(_, _)).WillRepeatedly(Return(language));
```

Comment:
@LitvinenkoIra , please create `MockMessageHelper& mock_message_helper_` in fixture and use it instead of `*(MockMessageHelper::message_helper_mock())`.
Also add `Mock::VerifyAndClearExpectations(&mock_message_helper_)` to `SetUp` and `TearDown` funcs